components/automate-ui/src/app/entities/orgs/org.actions.ts

```diff
@@ -0,0 +1,141 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Action } from '@ngrx/store';
+
+import { Org } from './org.model';
+
+export enum OrgActionTypes {
+  GET_ALL = 'ORGS::GET_ALL',
+  GET_ALL_SUCCESS = 'ORGS::GET_ALL::SUCCESS',
+  GET_ALL_FAILURE = 'ORGS::GET_ALL::FAILURE',
+  GET = 'ORGS::GET',
+  GET_SUCCESS = 'ORGS::GET::SUCCESS',
+  GET_FAILURE = 'ORGS::GET::FAILURE',
+  CREATE = 'ORGS::CREATE',
+  CREATE_SUCCESS = 'ORGS::CREATE::SUCCESS',
+  CREATE_FAILURE = 'ORGS::CREATE::FAILURE',
+  DELETE = 'ORGS::DELETE',
+  DELETE_SUCCESS = 'ORGS::DELETE::SUCCESS',
+  DELETE_FAILURE = 'ORGS::DELETE::FAILURE',
+  UPDATE = 'ORGS::UPDATE',
+  UPDATE_SUCCESS = 'ORGS::UPDATE::SUCCESS',
+  UPDATE_FAILURE = 'ORGS::UPDATE::FAILURE'
+}
+
+export interface OrgSuccessPayload {
+  org: Org;
+}
+
+export class GetOrgsForServer implements Action {
```

Comment:
Seems like it would be a little more consistent to use this:
```suggestion
export class GetOrgs implements Action {
```